src/irmin-mirage/irmin_mirage.mli

```diff
@@ -39,36 +39,40 @@ end
 
 module Git: sig
 
-  module Make
+  module type S_MAKER = functor
```

Comment:
Should probably just be called `MAKER`

Reply:
I think I agree. Here I was trying to be consistent with the `S_MAKER` type in `irmin-git`:
https://github.com/mirage/irmin/blob/master/src/irmin-git/irmin_git.mli#L83

but perhaps that should also just be a `MAKER`?